src/parameterizations/vertical/MOM_bulk_mixed_layer.F90

```diff
@@ -528,13 +528,15 @@ subroutine bulkmixedlayer(h_3d, u_3d, v_3d, tv, fluxes, dt, ea, eb, G, GV, US, C
         RmixConst = -0.5*CS%rivermix_depth * GV%g_Earth
         do i=is,ie
           TKE_river(i) = max(0.0, RmixConst * dSpV0_dS(i) * &
-                        (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j)) * S(i,1))
+                        (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j) + &
```

Comment:
Please add parentheses to specify the order of the sum of these 4 runoff terms, but here and later in lines 538-539.

Reply:
Addressed in 69220e9cebf8bfdade695c9e617fd6002d99c84a.